curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -20,7 +20,7 @@ You should nest a new anchor (`a`) element within the `p` element.
 assert($('p > a').length);
 ```
 
-The link's href value should be `https://freecatphotoapp.com`. You have either omitted the href value or have a typo.
+The link's href value should be `https://freecatphotoapp.com`. You have either omitted the `href` value or have a typo.
```

Comment:
Please reread through the issue more carefully because not all of the required changes have been met here.